lib/studies/reply/adaptive_nav.dart

```diff
@@ -665,6 +675,88 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
   }
 }
 
+class _BottomAppBarActionItems extends StatelessWidget {
+  const _BottomAppBarActionItems({@required this.drawerVisible})
+      : assert(drawerVisible != null);
+
+  final bool drawerVisible;
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<EmailStore>(
+      builder: (context, model, child) {
+        final onMailView = model.currentlySelectedEmailId >= 0;
+
+        return AnimatedSwitcher(
+          duration: const Duration(milliseconds: 350),
```

Comment:
It probably makes sense to extract the 350 to a const somewhere since I think you use it a few times throughout reply

Reply:
I'll revisit this in a new PR.